### DISCUSSION TITLE
Validate monorepo changes before deployment trigger

### DISCUSSION THREAD

**Discussion by itsmnthn** [vercel/community#1197] [Help] [answered]
The feature is to ignore deployment when there are no changes made in the dir that is being deployed on Vercel.

For example.
```bash
# Monorepo
-- monorepo
  -- packages
      -- pkg1
      -- pkg2
      -- pkg3 (Deployed on Vercel)
```

In above example making changes in `pkg1` or `pkg2` should not trigger `pkg3` deployment

**Comment by amyegan** [comment, marked as answer]
Hi @itsmnthn! Using `npx turbo-ignore` as the Ignored Build Step for your project will intelligently ignore changes to other projects. You can read more about it in [Intelligent ignored builds using Turborepo](https://vercel.com/changelog/intelligent-ignored-builds-using-turborepo) and in the docs on [using turbo-ignore](https://vercel.com/docs/concepts/monorepos/turborepo#using-turbo-ignore).
